QuickOSM/test/test_query_factory.py

```diff
@@ -600,7 +600,22 @@ def test_make(self):
             '<recurse type="down"/>\n    </union>\n    '
             '<print mode="body"/>\n</osm-script>\n'
         )
-        self.assertEqual(query.make(), expected)
+        self.assertEqual(query.make(False), expected)
+
+        query = QueryFactory(
+            query_type=QueryType.BBox, key='foo', value='bar')
+        self.assertIsNone(query.area)
+        expected = (
+            '[out:xml] [timeout:25];\n'
+            '(\n'
+            '      node["foo"="bar"]( {{bbox}});\n'
+            '      way["foo"="bar"]( {{bbox}});\n'
+            '      relation["foo"="bar"]( {{bbox}});\n'
+            ');\n'
+            '(._;>;);\n'
+            'out body;'
+        )
+        self.assertEqual(query.make(True), expected)
```

Comment:
```suggestion
        self.assertEqual(query.make(OverpassLanguage.Oql), expected)
```
Outside of context, it helps to have better code readability.